packages/agent/src/sync-manager.ts

```diff
@@ -239,7 +238,7 @@ export class SyncManagerLevel implements SyncManager {
 
     for (let job of pushJobs) {
       const [key] = job;
-      const [did, dwnUrl, watermark, messageCid] = key.split('~');
+      const [did, dwnUrl, _, messageCid] = key.split('~');
```

Comment:
@LiranCohen Is there a reason we're still using the watermark in the key?
In other words, why not delete it rather than assigning it to a value we don't use? (`_`)

Reply:
@frankhinek Yeah, so the events come in the order they should be inserted when doing a `EventsGet`. We then stick them in a LevelDB queue, to read here.  The queue is sorted by the key, so the watermark here is used to sort it in the order it should be applied.